botutils/botconfig/config.go

```diff
@@ -11,8 +11,34 @@ import (
 	"github.com/palantir/go-githubapp/githubapp"
 )
 
+//go:generate mockgen -destination os_mock_test.go -self_package github.com/solo-io/go-utils/botutils/botconfig -package botconfig_test github.com/solo-io/go-utils/osutils OsClient
+
 const (
 	DefaultBotCfg  = "/etc/solo-github-app/config.yml"
+
+	BotConfigEnvVar = "BOT_CONFIG"
+	WebhookSecretEnvVar = "WEBHOOK_SECRET"
+	PrivateKeyEnvVar = "PRIVATE_KEY_FILE"
+	IntegrationIdEnvVar = "INTEGRATION_ID"
+	InstallationIdEnvVar = "INSTALLATION_ID"
```

Comment:
Installation ID should come from the request, not from the env. having it in the config was a temporary thing while we figured out how to pass it through the build object

Reply:
Cool, going to remove the app config altogether -- it's bot specific. we will want to wire up build config at some point but don't have a solution for that right now